include/zephyr/debug/cpu_load.h

```diff
@@ -48,6 +49,23 @@ int cpu_load_get(bool reset);
  */
 void cpu_load_log_control(bool enable);
 
+/* Optional callback for cpu_load_cb_reg
+ *
+ * This will be called from the k_timer expiry_fn.
```

Comment:
```suggestion
 * This will be called from the k_timer expiry_fn used for periodic logging. CONFIG_CPU_LOAD_LOG_PERIODICALLY must be configured to a positive value.
```